docs/runtimes/application.ipynb

```diff
@@ -59,17 +62,36 @@
     "application = project.set_function(\n",
     "    name=\"my-vizro-dashboard\", kind=\"application\", requirements=[\"vizro\"]\n",
     ")\n",
-    "application.set_internal_application_port(add - your - port)\n",
-    "application.spec.command(add - command - string)\n",
-    "application.spec.args(add - command - args - list - of - strings)\n",
+    "application.set_internal_application_port(8050)\n",
+    "application.spec.command = \"gunicorn\"\n",
+    "application.spec.args = [\n",
+    "    \"<my-app>:<my-server>\",\n",
+    "    \"--bind\",\n",
+    "    \"0.0.0.0:8050\",\n",
+    "]\n",
+    "\n",
     "# Provide code artifacts\n",
     "application.with_source_archive(\n",
-    "    \"git://github.com/org/repo#my-branch\", pull_at_runtime=True\n",
+    "    \"git://github.com/org/repo#my-branch\", pull_at_runtime=False\n",
     ")\n",
+    "\n",
     "# Build the application image via MLRun and deploy the Nuclio function\n",
-    "# Optionally add mlrun and/or a base image\n",
+    "# Optionally add mlrun\n",
     "application.deploy(with_mlrun=False)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Reusing an already built reverse proxy image is done when:\n",
+    "1. Redeploying a function that built the reverse proxy once and has `application.status.container_image` enriched.\n",
+    "2. After it was built manually with `mlrun.runtimes.ApplicationRuntime.deploy_reverse_proxy_image()`.\n",
```

Comment:
"- Redeploying a function that built the reverse proxy once and has `application.status.container_image` enriched.\n",
    "- It was already built manually with `mlrun.runtimes.ApplicationRuntime.deploy_reverse_proxy_image()`.\n",